### DISCUSSION TITLE
Help for a few customizations

### DISCUSSION THREAD

**Discussion by alxgda** [hugo-fixit/FixIt#284] [Q&A] [answered]
Hello!

First, thank you for this theme! I started to use it for [my blog](https://blog.alexandregaida.com). I installed it as a Hugo submodule. I wanted to do some customizations, and I successfully did so for a few ones. Now, as I'm not an expert on CSS and Hugo templating, I'm requesting some help for the following:

- In the posts section, I would like the **full date shown for each post** (not just the month and date). I tried to do it but ended up with a strange year value. It would be nice if we could choose that.
- I think that it would be better if the **post update date would be shown in the beginning instead of the end,** because some people will not go until the end of the post if they think that it's outdated (especially for technical posts). I did not find how I could change the location of that info.
- I think that the theme is very similar to https://deepthought-theme.netlify.app. I would like to do this kind of homepage, with **a more thorough and wide textual description of who I am, some project I'm working on etc.**. Actually, we can only specify a subtitle. 
- I would like to be able to **change the avatar image size**. It's actually 8rem, I wanted to change it to 12 rem, but for reasons that I cannot understand, the overloading with the same file name in my repo (/assets/css/_page/_home.scss) doesn't work. My file is not taken into account. It's strange because I could successfully modify others parts of the theme by overloading the files.

Thank you for reading and for your help!

**Comment by Lruihao** [comment, marked as answer]
1. [time.Format](https://pkg.go.dev/time#Time.Format) in golang is `2006-01-02 15:04:05` that means `YYYY-MM-DD hh:mm:ss` like orther programming language. So for your problem, you can configure like this:
    ```toml
    [section]
      # ...
      # date format (month/day year)
      dateFormat = "01/02 2006"
    ```
2. This is a good suggestion, please look forward to the follow-up development.
    See https://github.com/hugo-fixit/FixIt/issues/285
3. You can create a `profile.html` located in `/layouts/partials/custom/` and write the long description in it. e.g. [profile.html](https://github.com/hugo-fixit/docs/blob/main/layouts/partials/custom/profile.html)
4. In simple terms, just changle the avatar image style in `_custom.scss` located in `/assets/css/_custom.scss`:
    ```scss
    .home {
      .home-profile {
        .home-avatar img {
          width: 12rem;
        }
      }
    }
    ```

**Reply by alxgda** [reply]
Thank you! I was able to successfully update my blog with your help!
